### DISCUSSION TITLE
Clarify gitlab runner docs

### DISCUSSION THREAD

**Discussion by viceice** [renovatebot/renovate#10488] [Request Help]
@HonkingGoose are you willing to help use to clarify and optimise our docs at https://gitlab.com/renovate-bot/renovate-runner ?

@rarkins Can also invite you to our slack channel for easier / private discussions. 🙃

**Comment by HonkingGoose** [comment]
@viceice What's the scope of the documentation? Is it only the current README or are there more files to change?

**Reply by viceice** [reply]
Currently only the readme. I also want some hints there to send users to discussion here for getting help.

So we only have feature / bug issues in gitlab

**Reply by HonkingGoose** [reply]
> @HonkingGoose are you willing to help use to clarify and optimise our docs at https://gitlab.com/renovate-bot/renovate-runner ?

Normally I'd say yes, but I already have enough projects on my hands right now. I don't have the time to work on this. So for now I'll have to decline the offer. But still, thank you for thinking of me! 😉 

I'm sorry that I cannot help you out with this right now. 😞

**Comment by rarkins** [comment]
@HonkingGoose is welcome to join our Slack at any time but I need an email address to invite, if they wish

**Reply by HonkingGoose** [reply]
I got the invite!
